app/src/main/java/org/fossasia/openevent/app/core/event/create/CreateEventFragment.java

```diff
@@ -79,10 +94,17 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
 
         setHasOptionsMenu(true);
 
-        binding.submit.setOnClickListener(view -> {
-            if (validator.validate())
-                getPresenter().createEvent();
-        });
+        if(eventId != -1) {
```

Comment:
Also, `eventId != -1` is checked repeatedly. It'd be better if this is changed to a logical boolean property like `isUpdateEvent` or better

Reply:
Actually i am using it because in case when update event is chosen, then an event id is passed as a parameter with intent as it is needed.I can use usUpdatEvent boolean but i will need to putExtra( ) a boolean along with the event id